tests/integration/generated/test_min_case_global_time_series_custom_chrysalis.cfg

```diff
@@ -34,6 +34,7 @@ years = "1985:1995:5",
 
 [global_time_series]
 active = True
+environment_commands = "source /gpfs/fs1/home/ac.forsyth2/miniforge3/etc/profile.d/conda.sh; conda activate zppy_interfaces_dev_pr_1_20241121v4"
```

Comment:
Presumably, this will be removed once there are shared environments that support zppy-interfaces?

Reply:
Well, realistically this will remain because we'll always want to test `zppy` with the latest `zppy-interfaces` code, not whatever the last release was. But yes, for `cfg`s written by users they could rely on Unified having `zppy-interfaces` in it.

Reply:
Or is this just a placeholder?

Reply:
Oh, that's the responsibility of `tests/integration/utils.py`. This file is auto-generated by that one. I currently only have a Chrysalis environment set up, so it's the only one defined.
![Screenshot 2024-11-22 at 11 27 46 AM](https://github.com/user-attachments/assets/41aaa653-8880-4a2b-a887-404ac522f011)

Reply:
Why is a file autogenerated by a script included in the repo at all?

Reply:
They originally weren't. Then @chengzhuzhang and I consciously decided to include them because users wanted examples that were fully filled out (i.e., not templated).

Reply:
Okay, good to know.  Doesn't this create a lot of distracting diffs in updates if you keep changing them?

Reply:
Yep, it's why I just collapse the `generated` subdirectory when I do reviews; and I also have to tell reviewers to ignore them.

Reply:
Okay, I'll count myself as informed, then.

Reply:
I also think it is a little odd to have machine and user specific path here, but don't know how to get around.

Reply:
Well we could use `$USER` to get my name out of it but ultimately whoever runs this cfg will need to create their own conda environment for the latest developments or else use Unified. 

I think maybe the best solution is to always keep the `environment_commands` lines on the `main` branch filled in as `<path to YOUR conda>; conda activate <environment YOU create>`.

Reply:
Ok, the commit https://github.com/E3SM-Project/zppy/pull/642/commits/dbd74f3130f96c6ddf31cc07cf536bd7c03c615e implements the idea I mentioned in the previous comment.

Reply:
It seems like it would be better to have some examples somewhere but not to update them very often.  This is the approach we use in MPAS-Analysis.

Reply:
The problem with that is I fear they will very quickly go out-of-date. This approach (the auto-generation of scripts from templates) updates everything together.